### ISSUE TITLE
Seems to fail to backup wireguard app

### ISSUE THREAD

**Issue by Napator** [YunoHost-Apps/borg_ynh#143]
### Describe the bug

Error during the backup scheduled daily : "Backup state: failed"
Seems to fail to backup wireguard app.

After backup i get 
"_auto_conf-2023-05-31_00:00
_auto_data-2023-05-31_00:01
_auto_borg-2023-05-31_00:01
_auto_freshrss-2023-05-31_00:02
_auto_nextcloud-2023-05-31_00:03
_auto_wallabag2-2023-05-31_00:05"

And also data in borgbase repo.

### Context

- Hardware: *Raspberry Pi 3 at home 
- YunoHost version: 11.1.20 (stable)
- I have access to my server: through the webadmin
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: yes
  - If yes, please explain: Baclup to BorgBase repository
- Using, or trying to install package version/branch: 1.1.16~ynh30

### Steps to reproduce
Configure distant backup to borgbase with no error at save : 
"Checking what changed in the new configuration...
Validating the new configuration...
Applying the new configuration...
'/home/yunohost.multimedia/.nobackup' wasn't deleted because it doesn't exist.
Configuration of borg completed
Config updated as expected"

### Expected behavior
Manage to finish backup

### Logs
https://paste.yunohost.org/tasixupexu

**Comment by Napator** [comment]
Tested manual local backup of wireguard successfully.

Tested also backup to borgbase with "exclude:wireguard" successfully.

It seems only the backup with of wireguard with borg is not working.

Thank you for your help.

**Comment by jorge-vitrubio** [comment]
this issue is related to this one https://github.com/YunoHost-Apps/wireguard_ynh/issues/70

but some of us didn't manage to use the "exclude:wireguard" so the yunohost borg keep trhowing errors every day

```
cat /var/log/yunohost/categories/operation/20230627-000322-backup_create.yml
args:
  apps:
  - wireguard
  description: null
  dry_run: false
  methods:
  - borg_app
  name: auto_wireguard
  output_directory: null
  system: null
ended_at: 2023-06-27 00:03:25.577427
error: '''/etc/wireguard/wg0.conf'' and ''/home/yunohost.backup/tmp/auto_wireguard/apps/wireguard/backup/etc/wireguard/wg0.conf''
  are the same file'
interface: cli
operation: backup_create
parent: null
related_to:
- - app
  - wireguard
started_at: 2023-06-27 00:03:22.479313
success: false
yunohost_version: 11.1.21.4
```

**Comment by CodeShakingSheep** [comment]
I can see that https://github.com/YunoHost-Apps/wireguard_ynh/issues/70 in Wireguard repository is closed. However, the issue still persists for me. Anyone else?

**Comment by jorge-vitrubio** [comment]
> I can see that [YunoHost-Apps/wireguard_ynh#70](https://github.com/YunoHost-Apps/wireguard_ynh/issues/70) in Wireguard repository is closed. However, the issue still persists for me. Anyone else?

have you tried this? -> https://github.com/YunoHost-Apps/wireguard_ynh/issues/70#issuecomment-1758668254

**Comment by CodeShakingSheep** [comment]
> > I can see that [YunoHost-Apps/wireguard_ynh#70](https://github.com/YunoHost-Apps/wireguard_ynh/issues/70) in Wireguard repository is closed. However, the issue still persists for me. Anyone else?
> 
> have you tried this? -> [YunoHost-Apps/wireguard_ynh#70 (comment)](https://github.com/YunoHost-Apps/wireguard_ynh/issues/70#issuecomment-1758668254)

Yes, I commented this line in Wireguard's backup script yesterday but the backup still failed last night.

**Comment by jorge-vitrubio** [comment]
> > > I can see that [YunoHost-Apps/wireguard_ynh#70](https://github.com/YunoHost-Apps/wireguard_ynh/issues/70) in Wireguard repository is closed. However, the issue still persists for me. Anyone else?
> > 
> > 
> > have you tried this? -> [YunoHost-Apps/wireguard_ynh#70 (comment)](https://github.com/YunoHost-Apps/wireguard_ynh/issues/70#issuecomment-1758668254)
> 
> Yes, I commented this line in Wireguard's backup script yesterday but the backup still failed last night.

you have to apply changes to cron by restartin ther service or even restarting the server will do

**Comment by CodeShakingSheep** [comment]
Yesterday I restarted my whole server, but the backup is still not successful. I commented this line in the backup script as suggested in the solution:

```
# Backing up specific config file, in case of it is not in /etc/wireguard
#ynh_backup --src_path="$(jq -r 'select(.config_file_path | startswith("/etc/wireguard/") | not) | .config_file_path' $install_dir/db/server/global_settings.json)" --not_mandatory
```

Any other ideas?

Also, if I understand correctly, instead of restarting the whole server I would need to restart `wg-quick@wg0` service, right?

**Comment by fflorent** [comment]
If I am correct, the issue is a duplicate of this one: https://github.com/YunoHost-Apps/wireguard_ynh/issues/70

Which is solved by this PR: https://github.com/YunoHost-Apps/wireguard_ynh/pull/96

Closing, ask for reopening it or open another issue if that's not the case
